modules/ROOT/pages/to-search-with-graph-api.adoc

```diff
@@ -39,15 +39,15 @@ See also https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97b
 
 == Graph API Access URL
 
-You can access the Graph API at the `+https://anypoint.mulesoft.com/graph/api/v1/graphql+` URL. To search
+You can access the Graph API at the `+https://anypoint.mulesoft.com/graph/api/v1/graphiql+` URL. To search
```

Comment:
The change is not correct. That's the URL for the **graphiql** interface (a nice UI on top of the API). The actual API URL was correct. We could add the information for the **graphiql** interface, but let's not change the API URL.

Reply:
Thank you @mcampo ! @robertpliu if you'd like to update this PR let me know. Otherwise I'll close this one Monday.

Reply:
The https://anypoint.mulesoft.com/graph/api/v1/graphql returns:
HTTP method not allowed, supported methods: POST
<img width="993" alt="Screen Shot 2019-07-12 at 5 43 27 PM" src="https://user-images.githubusercontent.com/47807427/61164732-ded07d80-a4cc-11e9-85b0-3445210264fc.png">

the https://anypoint.mulesoft.com/graph/api/v1/graphiql/ returns the correct GraphiQL screen:
<img width="993" alt="Screen Shot 2019-07-12 at 5 43 38 PM" src="https://user-images.githubusercontent.com/47807427/61164759-0d4e5880-a4cd-11e9-942e-c2e43f4bb379.png">

Reply:
Ok, now I understand https://anypoint.mulesoft.com/graph/api/v1/graphql is to be used for command line interface so there's no need to change the documentation. However, do we have any documentation on the browser interface for https://anypoint.mulesoft.com/graph/api/v1/graphiql/ ?

Reply:
I was not able to find documentation on graphiql in the Exchange or Connectors docs.